### PR TITLE
Unpin Black version and apply formatting changes

### PR DESCRIPTION
**Pull request by pwt**
This removes the version pin for Black used in the checks, and updates the formatting of all affected files.

### CODE REVIEW

**Review by KennethNielsen — CHANGES_REQUESTED** [review]
Just a single comment about the required black version otherwise good to go

File: requirements-dev.txt [hunk 1]
```diff
@@ -8,4 +8,4 @@ pylint == 2.5
 coveralls
 pytest-cov<2.6.0
 wheel
-black==19.10b0; python_version >= "3.6" and implementation_name == "cpython"
+black; python_version >= "3.6" and implementation_name == "cpython"
```
[the remark this answers is not in the document]
Comment (pwt) [reply requirements-dev.txt]: Sounds good. Done.

**Comment by pwt** [comment]
Turns out that Black doesn't play well with Pip's '>=' versioning syntax unless the 'b0' is included in the Black version string. It does then successfully pick up the latest version, until Black changes its versioning scheme.

**Review by KennethNielsen — APPROVED** [review]
All good.

**Pull request merged**